build.psm1

```diff
@@ -4,6 +4,79 @@ $script:TestModulePathSeparator = [System.IO.Path]::PathSeparator
 
 $dotnetCLIChannel = "preview"
 $dotnetCLIRequiredVersion = "2.0.0-preview2-006502"
+$tagsUpToDate = $false
+
+function Sync-PSTags
```

Comment:
Can you please add a comment to explain why we need to run `Sync-PSTags` before getting the version (tag) and commit ID? It will help people to understand.